src/api/spec/controllers/project/public_key_controller_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Project::PublicKeyController, type: :controller, vcr: true do
```

Comment:
As you don't add cassettes do you really need vcr: true?
https://github.com/vcr/vcr